File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Workflows/WorkflowAcceptanceModel.ts
import RoleModel = require('App/Models/Commons/RoleModel');
import AcceptanceStatus = require('App/Models/Workflows/AcceptanceStatus');

interface WorkflowAcceptanceModel {
    Owner: string;
    Executor: string;
    ProposedRole: RoleModel;
    Status: AcceptanceStatus;
    AcceptanceDate: Date;
    AcceptanceReason: string;
}
export = WorkflowAcceptanceModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Workflow/WorkflowAcceptanceModel.cs
using System;

namespace VecompSoftware.DocSuiteWeb.Model.Workflow
{
    public class WorkflowAcceptanceModel
    {
        #region [ Properties ]

        public string Owner { get; set; }

        public string Executor { get; set; }

        public WorkflowRole ProposedRole { get; set; }

        public AcceptanceStatus Status { get; set; }

        public DateTimeOffset? AcceptanceDate { get; set; }

        public string AcceptanceReason { get; set; }
        #endregion
    }
}
